.github/workflows/build.yaml

```diff
@@ -20,10 +20,6 @@ jobs:
     - run: |
         # Compile the protos to make sure they're valid
         ./compile.sh
-        
```

Comment:
Why not replace this with the new generator? Doesn't this leave us open to things being out of date?

Reply:
I answer this question in the PR description.

A different PR that updates the CI (which is currently outdated).
https://github.com/googleapis/google-cloudevents/pull/117